.gitignore

```diff
@@ -0,0 +1,2 @@
+test.md
```

Comment:
- [ ] [OPTIONAL] this file seems to be from another project. you can safely remove this "test.md" from your .gitignore if you don't have that file on your workspace.